invenio/ext/sqlalchemy/testsuite/test_composite_dicts.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
```

Comment:
@dset0x `invenio/ext/sqlalchemy/testsuite/` is not part of test suite lookup registry. Please move the file to: `invenio/testsuite/test_ext_sqlalchemy_utils.py`